src/AspNetCoreModuleV2/AspNetCore/ServerErrorHandler.h

```diff
@@ -3,21 +3,31 @@
 
 #pragma once
 #include "requesthandler.h"
+#include "file_utility.h"
+
+extern HINSTANCE g_hServerModule;
 
 class ServerErrorHandler : public REQUEST_HANDLER
 {
 public:
-    ServerErrorHandler(IHttpContext &pContext, HRESULT hr) : m_pContext(pContext), m_HR(hr)
+
+    ServerErrorHandler(IHttpContext &pContext, HRESULT hr, bool disableStartupPage, int page)
```

Comment:
Why not merge StartupExceptionHandler and this?

Reply:
I'll propagate the module to the handlers and then make that change.